mods/default/mapgen.lua

```diff
@@ -576,8 +576,29 @@ function default.register_ores()
 		},
 		-- Only where default:dirt is present as surface material
 		biomes = {"taiga", "snowy_grassland", "grassland", "coniferous_forest",
-			"deciduous_forest", "deciduous_forest_shore", "savanna", "savanna_shore",
-			"rainforest", "rainforest_swamp", "floatland_coniferous_forest"}
+				"deciduous_forest", "deciduous_forest_shore", "rainforest",
+				"rainforest_swamp", "floatland_coniferous_forest"}
+	})
+
+	minetest.register_ore({
```

Comment:
This was good and thorough of you to add this, but i prefer to not add another blob ore.
Dirt blobs are just added to biomes that have normal dirt, savanna simply no longer has normal dirt, just like tundra has permafrost dirt.

Reply:
done, and rebased the branch successfully (I think)